api/client.js

```diff
@@ -1,5 +1,9 @@
 import { Client, Intents } from "discord.js";
 
+import logger from "../logging.js";
+
+const childLogger = logger.child({ component: "discord client" });
```

Comment:
why do we need a child logger here?
How is this different from the other places?

Reply:
It adds more context/metadata to the log. The logging looks like this:
```
{"component":"discord client","level":"debug","message":"ready","timestamp":"2022-11-05T00:00:30.106Z"}
```

The other loggers are not very clear to be a separate component, in my opinion